content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,117 @@
+
+
+Our core focus for the 1.9 release was to improve the [Day 2 operations](https://dzone.com/articles/defining-day-2-operations)
+for users running Istio in production. Building upon the feedback collected from the User Experience Working
+Group, we wanted to improve the stability and overall upgrade experience for our users. A key aspect of
+stability is publishing accurate [feature status](https://istio.io/latest/about/feature-stages/) for Istio core APIs
+and features, and progressing their stability to enable our users to utilize Istio's capabilities with confidence,
+which was another focus for Istio 1.9 release.
+
+Our core focus for the 1.9 release was to improve the [Day 2 operations](/docs/reference/glossary/#D) for users
+running Istio in production. Building upon the feedback collected from the User Experience Working
+Group, we wanted to improve the stability and overall upgrade experience for our users. A key aspect of
+stability is publishing accurate [feature status](https://istio.io/latest/about/feature-stages/) for Istio core APIs
+and features, and progressing their stability to enable our users to utilize Istio's capabilities with confidence, which
+was another focus for Istio 1.9 release.
+
+Thank you to our users who participated in user experience surveys and empathy session to ensure Istio 1.9 is our most stable release to date.
+
+We will be publishing our plans to continue improving the Day 2 operations as a focus area for Istio 2021 roadmap in an upcoming blog .
+
+Here are some highlights for this release:
+
+## Feature promotion
+
+### Virtual Machine (VM) expansion promoted to Beta
+
+Enabling workloads running in VMs to be part of the Istio service mesh, and being able to apply consistent policy and telemetry across containers and VMs has always been a focus of the Istio community.  We have continued improving the stability, testing and documentation for VM expansion, and are happy to announce that in Istio 1.9 we have promoted this feature to Beta.
+
+Here's a list of supporting documents which you can follow to easily expand your Istio service mesh to include VMs:
+
+* [Virtual Machine Installation](/docs/setup/install/virtual-machine/) to get started.
+* [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) to learn about the high level architecture of Istio's virtual machine integration.
+* [Debugging Virtual Machines](/docs/ops/diagnostic-tools/virtual-machines/) to learn more about troubleshooting issues with virtual machines.
+* [Bookinfo with a Virtual Machine](/docs/examples/virtual-machines/) to learn more about connecting virtual machine workloads to Kubernetes workloads.
+
+VM support will continue to be a high priority for us in 2021, and you can expect us to see more feature
+graduation in upcoming releases.
+
+### External Istiod promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
```

Comment:
The external control plane deployment model introduced in 1.8 gives you the ability to run the Istio control plane (istiod) on a different cluster than where your mesh workload is running. This creates a clear separation of ownership between the mesh operators and the mesh admins allowing for more robust installations. We're promoting this to Beta and encourage you to try it!